react/ecmascript/02 - promises/index.js

```diff
@@ -1,9 +1,16 @@
 // Hint: use setInterval, create a new Promise and measure time with Date.now()
 
-export function delay() {
-
+export function delay(time) {
+  const wait = ms =>
```

Comment:
You are not using `ms` and you don't need the function `wait`. You can just return the promise